docs/structured-streaming-programming-guide.md

```diff
@@ -1777,7 +1779,7 @@ Here is the compatibility matrix.
     <td style="vertical-align: middle;">Update operation mode</td>
     <td style="vertical-align: middle;">Update</td>
     <td style="vertical-align: middle;">
-      Aggregations not allowed after <code>flatMapGroupsWithState</code>.
+      Aggregations not allowed in a query with <code>flatMapGroupsWithState</code>.
```

Comment:
Aggregations are not just disallowed after `flatMapGroupsWithState` in in update mode. It is disallowed in the query.